electron/src/menu/system.ts

```diff
@@ -437,26 +439,25 @@ export const createMenu = (isFullScreen: boolean): Menu => {
   }
 
   processMenu(menuTemplate, locale.getCurrent());
-  menu = Menu.buildFromTemplate(menuTemplate);
-
-  return menu;
+  return Menu.buildFromTemplate(menuTemplate);
 };
 
 export const registerShortcuts = (): void => {
   // Global mute shortcut
-  globalShortcut.register('CmdOrCtrl+Alt+M', () =>
+  const muteShortcut = 'CmdOrCtrl+Alt+M';
+  logger.info(`Registering global mute shortcut "${muteShortcut}" ...`);
+  globalShortcut.register(muteShortcut, () =>
     WindowManager.sendActionToPrimaryWindow(EVENT_TYPE.UI.SYSTEM_MENU, EVENT_TYPE.CONVERSATION.TOGGLE_MUTE),
   );
 
   // Global account switching shortcut
-  const switchAccountShortcut = ['CmdOrCtrl', 'Super'];
```

Comment:
Let's not switch accounts when a user presses <kbd>Win</kbd>+<kbd>1</kbd> if they can use <kbd>Ctrl</kbd>+<kbd>1</kbd>.